resources/zib-AddressInformation.xml

```diff
@@ -0,0 +1,339 @@
+            <profile value="http://hl7.org/fhir/StructureDefinition/iso21090-ADXP-buildingNumberSuffix"/>
+         </type>
+         <mapping>
+            <identity value="zib-addressinformation-v1.1-2020EN"/>
+            <map value="NL-CM:20.5.11"/>
+            <comment value="HouseNumberLetter"/>
+         </mapping>
+         <mapping>
+            <identity value="zib-addressinformation-v1.1-2020EN"/>
+            <map value="NL-CM:20.5.10"/>
+            <comment value="HouseNumberAddition"/>
+         </mapping>
+      </element>
+      <element id="Address.line.extension:houseNumberIndication">
+         <path value="Address.line.extension"/>
+         <sliceName value="houseNumberIndication"/>
+         <short value="HouseNumberIndication"/>
+         <definition value="The indication used for addresses which do not consist of the usual street name and house number."/>
+         <alias value="AanduidingBijNummer"/>
+         <max value="1"/>
+         <type>
+            <code value="Extension"/>
+            <profile value="http://hl7.org/fhir/StructureDefinition/iso21090-ADXP-additionalLocator"/>
+         </type>
+         <constraint>
+            <key value="houseNumberIndication"/>
+            <severity value="error"/>
+            <human value="Must have value 'by' or 'to'"/>
+            <expression value="value in ('by' | 'to')"/>
+            <source value="http://nictiz.nl/fhir/StructureDefinition/zib-AddressInformation"/>
+         </constraint>
+         <mapping>
+            <identity value="zib-addressinformation-v1.1-2020EN"/>
+            <map value="NL-CM:20.5.9"/>
+            <comment value="HouseNumberIndication"/>
+         </mapping>
+      </element>
+      <element id="Address.line.extension:additionalInformation">
+         <path value="Address.line.extension"/>
+         <sliceName value="additionalInformation"/>
+         <short value="AdditionalInformation"/>
+         <definition value="Extra information such as the building name, building number, entrance, route number."/>
+         <alias value="AdditioneleInformatie"/>
+         <max value="1"/>
+         <type>
+            <code value="Extension"/>
+            <profile value="http://hl7.org/fhir/StructureDefinition/iso21090-ADXP-unitID"/>
+         </type>
+         <mapping>
+            <identity value="zib-addressinformation-v1.1-2020EN"/>
+            <map value="NL-CM:20.5.7"/>
+            <comment value="AdditionalInformation"/>
+         </mapping>
+      </element>
+      <element id="Address.city">
+         <path value="Address.city"/>
+         <short value="PlaceOfResidence"/>
+         <definition value="A geographically determined area which forms part of the municipal area. For Dutch places of residence, preferably use the name from the GBA, table 33 (OID: 2.16.840.1.113883.2.4.6.14)."/>
+         <comment value="For Dutch places of residence, preferably use the name from the GBA, table 33 (OID: 2.16.840.1.113883.2.4.6.14)."/>
```

Comment:
Dit comment staat ook in definition. Bewust?

Reply:
Hetzelfde geldt voor Address.district

Reply:
en Address.postalCode

Reply:
Dit staat zo in de zib Definitions: https://zibs.nl/wiki/AddressInformation-v1.1(2020EN). Zou je voorstellen het weg te halen omdat het wellicht verwarrend werkt?